pyqtgraph/functions.py

```diff
@@ -1903,15 +1903,15 @@ def _arrayToQPath_all(x, y, finiteCheck):
     numchunks = (n + chunksize - 1) // chunksize
     minchunks = 3
 
+    backfill_idx = None
+    if finiteCheck and not all_isfinite:
```

Comment:
nit-pick part2: we can combine this with the previous if-statement too? 

```python
backfill_idx = None
if finiteCheck:
    isfinite = np.isfinite(x) & np.isfinite(y)
    all_isfinite = np.all(isfinite)
    if not all_isfinite:
        backfill_idx = _compute_backfill_indices(isfinite)
```

Sorry, I should have specified this on the last comment!